user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+   "Activities" section on your User page.
+   
+   |
+   
+   |EmailTo|   
+   
+   |
+   
+- **Phone number**: is displayed to other users among your contact details and will be used as a default value to log a 
+  call related to you or call you via integrated phone applications, such as 
+  :ref:`Google Hangouts <user-guide-hangouts>`.
+
+  |
+  
+  |Phone|
+  
+  |
+  
+- **Email signature**: The signature may be added to any email you write in OroCRM. Your organization settings define 
+  whether the signature will be added automatically or manually. 
+
+
+.. _user-guide-getting-started-profile-password:
+
+Change Your Password
+^^^^^^^^^^^^^^^^^^^^
+
+To change your password, go to the *"Password"* section of the Edit page. 
+
+|
+
+.. image:: ./img/intro/user_edit_password.png
+
+|
+
+You have to:
+
+- Type in your current password
+
+- Type in the new password
+
+- Confirm the new password
+
+
+.. note::
+
+    If you are not using your Google account, nor your corporate-wide credentials, it is strongly recommended to change 
+    your password after the first log-in.
+
+
+.. _user-guide-intro-log-in-edit-profile-synchronize:
+
+Synchronize Your Mailbox with OroCRM
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Emails can be sent from user to user within OroCRM and outside OroCRM using internal OroCRM SMTPserver. However, most 
+OroCRM users already have some external mailbox used for work-related purposes. You can synchronize this mailbox with 
+your mailbox in OroCRM in the *"Email synchronization settings"* section.
+
+|
+
+.. image:: ./img/intro/email_sync_1.png
+
+|
+
+To synchronize your existing mailbox with your mailbox in OroCRM, go the the *"Email synchronization settings"* section.
+
+OroCRM can be synchronized with any IMAP/SMTP servers. A dedicated *"Gmail"* synchronization is available to simplify 
+synchronization with Gmail-based accounts. 
+If synchronization with an IMAP server has been defined, all the emails from synchronized folders of the external 
+mailbox will be available to you in *"My Emails"* in OroCRM. If If synchronization with an SMTP server has been defined, 
+all the emails sent from OroCRM will be available in the external mailbox.
+Please note, that if SMTP synchronization has been enabled, your addressee will always see the mailbox address as the 
+*"from"* field value.
+
+.. note::
+
+    If no synchronization has been enabled, emails received from other OroCRM users will appear in the *"Activity"* 
+    section of the *"My User"* page but not in *"My Emails"*. Emails from a synchronized mailbox can be reached from the 
+    both.
+
+Synchronize with any IMAP/SMTP Server
+"""""""""""""""""""""""""""""""""""""    
+
+The functionality can be used to synchronize any IMAP/SMTP server with your mailbox in OroCRM. 
+
+- Select the Account Type - *"Other"* (if available). If this is an only option enabled for the system, the selector 
+  won’t be displayed - skip the step.
```

Comment:
Edit: Select the Account Type - *"Other"* (if available). If this is the only option enabled for the system, the selector won’t be displayed, and you should just skip this step.